DataStructures/DynamicArray/DynamicArray.java

```diff
@@ -40,7 +40,7 @@ public DynamicArray() {
    * @return int the new capacity of the array
    */
   public int newCapacity() {
-    this.capacity *= 2;
+    this.capacity *= 1.8;
```

Comment:
Please add proper PR description of why this change is needed in current implementation.